buildpacks/nodejs-engine/node_runtime_metrics/metrics_collector.cjs

```diff
@@ -0,0 +1,243 @@
+/**
```

Comment:
I think this is the same code as https://github.com/heroku/heroku-buildpack-nodejs/blob/main/metrics/metrics_collector.cjs? Any thoughts on how we could avoid having two copies that we'd need to keep in sync?

Reply:
Assuming we want to keep them in sync? 😄 

But, considering that the previous metrics plugin was remained [(mostly) unmodified](https://github.com/heroku/heroku-nodejs-plugin/commits/main/src/monitor.js) during the several years it was in use, I don't imagine this will be changed that often. And the cost of manual copying is low for a single script with zero dependencies. Setting up a dedicated repository and publishing process seems unnecessary. 

Maybe a simple download & copy script could be added if we find we are making lots of changes?

Reply:
> I don't imagine this will be changed that often.

I agree.

> Setting up a dedicated repository and publishing process seems unnecessary.

Yeah, I agree there. Maybe one location could be authoritative (the CNB?) and the other (classic) copy could occasionally be synced? We're already doing something similar with the inventory management tooling.

I'm not hung up on it for this PR, though. We can cross that bridge when we need to update this plugin.